src/cart/src/cart/crt_rpc.c

```diff
@@ -1392,15 +1392,15 @@ crt_rpc_inout_buff_init(struct crt_rpc_priv *rpc_priv)
 static inline void
 crt_common_hdr_init(struct crt_rpc_priv *rpc_priv, crt_opcode_t opc)
 {
-	uint32_t	xid;
+	uint32_t	rpcid;
 
-	xid = atomic_fetch_add(&crt_gdata.cg_xid, 1);
+	rpcid = atomic_fetch_add(&crt_gdata.cg_rpcid, 1);
```

Comment:
should we wrap it?   otherwise, won't the rpcid change over time?

or could make cg_procid a constant 16 bit field and have cg_instance get incremented (and let it wrap) and then combine them here to the rpcid